Makefile

```
@@ -297,9 +300,26 @@ go-build-linux: $(BINARIES)
 run: docker-build kind-cluster kind-load kind-deploy #HELP Build the operator-controller then deploy it into a new kind cluster.
 
 .PHONY: docker-build
-docker-build: build-linux #EXHELP Build docker image for operator-controller with GOOS=linux and local GOARCH.
+docker-build: build-linux build-catalogd #EXHELP Build docker image for operator-controller with GOOS=linux and local GOARCH.
 	$(CONTAINER_RUNTIME) build -t $(IMG) -f Dockerfile ./bin/linux
 
+
+ifeq ($(origin IMAGE_CATALOG_REPO), undefined)
+IMAGE_REPO := quay.io/operator-framework/catalogd
+endif
+export IMAGE_REPO
+
+ifeq ($(origin IMAGE_CATALOG_TAG), undefined)
+IMAGE_TAG := devel
+endif
+export IMAGE_TAG
+
+IMAGE_CATALOG := $(IMAGE_CATALOG_REPO):$(IMAGE_TAG)
+
+.PHONY: build-catalogd
+build-catalogd: ## Build the catalogd project with specified IMAGE_TAG and IMAGE_REPO
+	IMAGE=$(IMAGE_CATALOG) $(MAKE) -C catalogd build-container
+
```

Do we need all of this? I wonder if we can do this instead:

```suggestion
docker-build: build-linux #EXHELP Build docker image for operator-controller and catalogd with GOOS=linux and local GOARCH.
 	$(CONTAINER_RUNTIME) build -t $(IMG) -f Dockerfile ./bin/linux
 	$(MAKE) -C catalogd build-container
```

---

If we call only $(MAKE) -C catalogd build-container, the image doesn’t build as expected. :-(
To make it work, we need to pass IMAGE=$(IMAGE_CATALOG) $(MAKE) -C catalogd build-container.

To address this, I’ve replicated the relevant parts from the other Makefile to ensure the repo:tag builds correctly.
We can pass the repo:tag directly instead like ` IMAGE=$(quay.io/operator-framework/catalogd:devel) $(MAKE) -C catalogd build-container` for now

---

This should be addressed when we combine the Makefiles, manifests, and other configuration files.
Question: Once combined, will we consolidate into a single manager, or will we retain separate managers for catalogd and operator-controller?